dashboard/app/models/sections/section.rb

```diff
@@ -304,6 +304,8 @@ def summarize(include_students: true)
       restrict_section: restrict_section,
       code_review_enabled: code_review_enabled?,
       is_assigned_csa: assigned_csa?,
+      # this will be true when we are in emergency mode and this section has any script assigned other than CSP or CSD
```

Comment:
It might be more accurate to say that this will be true for CSF and HoC courses?  (Even more precisely, it is the set of scripts returned [here](https://github.com/code-dot-org/code-dot-org/blob/746d5b11a6b879c19dbbc246430e19560aacda1c/lib/cdo/script_config.rb#L24-L30).)

Reply:
+1 to this suggestion because making the comment more specific will be more future-proof

Reply:
Thank you! This is definitely better!